pkg/auth/oauth/handlers/empties.go

```diff
@@ -25,11 +25,11 @@ func (EmptySuccess) AuthenticationSucceeded(user user.Info, state string, w http
 type EmptyError struct{}
 
 func (EmptyError) AuthenticationError(err error, w http.ResponseWriter, req *http.Request) (bool, error) {
-	glog.V(4).Infof("AuthenticationError: %v", err)
+	glog.Errorf("AuthenticationError: %#v", err)
```

Comment:
Why print the details the error instead of just the message?  That's what we do elsewhere.